Task-17/index.js

```diff
@@ -0,0 +1,16 @@
+function count() {
+    var counter = 0;
+    
+    return function (a) {
+        counter += a;
+        console.log(counter);
+    };
+}
+
+const sum = count()
+
+
+sum(2);
+sum(6);
+sum(17);
+sum(45);
```

Comment:
Внутрішня функція повинна повертати результат а не виводити його в консоль
Приберіть виведення в консоль в середені функціії, та зробіть так, щоб код нижче виводив значення а не `undefined`

```suggestion
console.log(sum(2));
console.log(sum(6));
console.log(sum(17));
console.log(sum(45));
```